packages/debug/src/browser/debug-session-connection.ts

```diff
@@ -290,6 +290,8 @@ export class DebugSessionConnection implements Disposable {
                 this.onDidCustomEventEmitter.fire(event);
             }
         } else {
+            // TODO: Does this branch even make sense?
+            // @ts-expect-error TS2559
```

Comment:
This one should go away with @tsmaeder's work on the debug system.

Reply:
Good to know, but I still need to keep this here for now.